samples/apps/bookinfo/bookinfo.yaml

```diff
@@ -156,7 +184,57 @@ spec:
         - containerPort: 9080
 ---
 ##################################################################################################
-# Productpage service
+# Mysql db services
+# credentials: root/password
+##################################################################################################
+apiVersion: v1
+kind: Secret
+metadata:
+  name: mysql-credentials
+type: Opaque
+data:
+  rootpasswd: cGFzc3dvcmQ=
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: mysqldb
+  labels:
+    app: mysqldb
+spec:
+  ports:
+  - port: 3306
+    name: mysql
+  selector:
+    app: mysqldb
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
```

Comment:
What I meant is to run the mysqldb service and deployment outside of the mesh, that is, put them into a separate yaml file and apply without `kube-inject`. Ratings v3 will still use the sidecar, it will just be calling an ordinary mysqldb service instead of an istiofied one.